src/components/Header/Header.jsx

```diff
@@ -3,23 +3,26 @@ import PropTypes from 'prop-types'
 import cx from 'classnames'
 import { ThemeContext } from 'components/useTheme'
 import styles from 'components/Header/Header.styl'
+import CozyTheme from 'cozy-ui/transpiled/react/CozyTheme'
 
 class Header extends React.PureComponent {
   render() {
     const { children, className, theme, fixed } = this.props
 
     return (
-      <ThemeContext.Provider value={theme}>
-        <div
-          className={cx(
-            styles[`HeaderColor_${theme}`],
-            { [styles.HeaderFixed]: fixed },
-            className
-          )}
-        >
-          {children}
-        </div>
-      </ThemeContext.Provider>
+      <CozyTheme variant="inverted">
+        <ThemeContext.Provider value={theme}>
```

Comment:
Do we still need the ThemeContextProvider? This is not the goal of the new `CozyTheme` ?